core/src/main/resources/org/apache/spark/ui/static/historypage-template.html

```diff
@@ -75,26 +75,6 @@
       </th>
   </thead>
   <tbody>
-  {{#applications}}
-    <tr>
-      {{#attempts}}
-      <td {{#hasMultipleAttempts}}style="background-color:#fff"{{/hasMultipleAttempts}}>{{version}}</td>
```

Comment:
Does this retain the logic about `hasMultipleAttempts`? I didn't see it below.

Reply:
No, It was not retaining this part of logic. 

Updated the code to apply same idea on first 3 columns (version, id and name) from datatables side.

Reply:
Hm I'm still not seeing this logic?